docs/quickstart.rst

```diff
@@ -75,9 +75,8 @@ Using root likely will be required if you want to backup files of other users
 or the operating system. If you only back up your own files, you neither need
 nor want to use root.
 
-Avoid to create a mixup of users and permissions in your repository (or cache).
-
-This can easily happen if you run borg using different user accounts (e.g. your
+Mixup of users and permissions in your repository (or cache) is unwanted.
+It will easily happen if you run borg using different user accounts (e.g. your
 non-privileged user and root) while accessing the same repo.
```

Comment:
I don't see the improvement here, can you explain? Are you a native english speaker?

Also: "will happen" is too strong as it also depends on some circumstances.

Reply:
> Avoid to create a mixup of users and permissions in your repository (or cache).

is unclear as it is not clear what is supposed to be avoided.

And in addition "Avoid to create a mixup of" is weird, as avoiding this will prevent mixup - not cause it.

"Are you a native english speaker?" no

Reply:
"Avoid to do something" is a recommendation not to do something, so it is clear (IMHO) what to avoid.

How exactly this happens is explained in the next sentence, referring back using "This".

Reply:
As we are both not native speakers, it would be good if a native speaker could improve the phrasing here, if necessary.

Reply:
> "Avoid to do something" is a recommendation not to do something, so it is clear (IMHO) what to avoid.

I think that "Avoid creating a mixup of users" would be correct if such structure was intended (but I am not a native speaker and so on).

Reply:
Not sure if that really expresses that (or if you are just "reading between the lines" here).

Maybe one could also say "Be careful not to create a mixup..." and that would express better that it could happen accidentally if one is not careful.

Reply:
"Avoid to create" is not correct English.   I also don't like the "a mixup of" part.  Looking at the FAQ item more broadly, it seems too long winded.  The current sentence could say "To avoid permissions issues, always access the repository using the same user account."  And this sentence should probably be the first one in the FAQ answer, followed by information about which account to use (root vs user) and how to achieve it (local vs ssh).

Another comment is that having every sentence as its own paragraph is unusual.  Some could be merged.

Reply:
done